src/test/java/org/eclipse/yasson/Assertions.java

```diff
@@ -0,0 +1,67 @@
+package org.eclipse.yasson;
+
+import static org.junit.Assert.fail;
+
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import javax.json.bind.JsonbException;
+
+public class Assertions {
```

Comment:
I am thinking about updating JUnit to version 5 instead of adding this class. As far as I know JUnit 5 should be able to handle this. What do you think? It might be some work to do, but I would say it is worth it to do. (I am not saying that you should update it by yourself.)

Reply:
@Verdent yea I'm all for JUnit 5, it's actually what inspired this helper class for now. Lets handle that separately from this PR though.
I assume you have no other objections to this PR and it's good to merge?

Reply:
Yes, I would say it looks good.